### DISCUSSION TITLE
Advice on loading battery

### DISCUSSION THREAD

**Discussion by rploeg** [WillCodeForCats/solaredge-modbus-multi#223] [General]
Hi,

A great add-on and great last update. I want to control my battery, but want to see your opinion. During the day I want to use my current setting 'Maximize self consumption'. 

But during the night my energy cost per kw/h is way lower than during the day. So I want to load my battery (especially now in the winter). Is it possible with the storage control in your add-on to enable loading the battery at a certain time and stop and set again he 'Maximize self consumption' option?

**Comment by Ashpork** [comment]
I have an HA automation in place that helps me achieve thIS.My Storage control mode is set to remove control. Storage command mode and storage default mode is MSC. I then use and automation to change storage command mode to charge from Solar and PV. Then set storage timeout to be the number of seconds I want the charge from Solar and of to stop and revert back to default mode.Sent from my iPhoneOn 1 Jan 2023, at 11:57, Remco Ploeg ***@***.***> wrote:﻿
Hi,
A great add-on and great last update. I want to control my battery, but want to see your opinion. During the day I want to use my current setting 'Maximize self consumption'.
But during the night my energy cost per kw/h is way lower than during the day. So I want to load my battery (especially now in the winter). Is it possible with the storage control in your add-on to enable loading the battery at a certain time and stop and set again he 'Maximize self consumption' option?
